leshan-server-cf/src/main/java/org/eclipse/leshan/server/californium/impl/LwM2mResponseBuilder.java

```diff
@@ -352,7 +352,7 @@ private LwM2mNode decodeCoapResponse(final LwM2mPath path, final Response coapRe
         LwM2mNode content;
         try {
             // get content format
-            ContentFormat contentFormat = null;
+            ContentFormat contentFormat = ContentFormat.TEXT;
```

Comment:
this seems to be the right place to deal with a missing content format. However, I am not sure that assuming TEXT is the right strategy.